macros/FirefoxSidebar.ejs

```diff
@@ -320,44 +320,56 @@ const text = mdn.localStringMap({
 
 <section class="Quick_links" id="Quick_Links">
   <ol>
-    <li><a href="#"><%=text["Firefox_developer_release_notes"]%></a>
-      <ol>
-        <li><a href="<%=baseURL%>Firefox/Releases"><%=text["Firefox_developer_release_notes"]%></a></li>
-      </ol>
+    <li class="toggle">
```

Comment:
``{{FirefoxSidebar}}`` also looks correct 👍. Here is it on https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/62:

<img width="505" alt="pr 789 firefoxsidebar" src="https://user-images.githubusercontent.com/286017/45334450-73cab900-b540-11e8-913a-30eff6743f3a.png">